packages/typedoc-plugin-appium/lib/model/types.ts

```diff
@@ -0,0 +1,37 @@
+import {Comment, DeclarationReflection, ProjectReflection} from 'typedoc';
+import {CommandInfo} from './command-info';
+import {CommandsReflection} from './reflection';
+
+/**
+ * I don't think we allow other HTTP methods?
```

Comment:
these are the only 3 that are part of the w3c spec. i don't think we want to open the door to others

Reply:
k